docs/index.html

```diff
@@ -184,6 +196,74 @@ <h3 class="panel-title"><span class="operation-name">POST</span> <strong>/notifi
                 </div>
             </div>
         
+        <span id="path--product"></span>
+            <div id="operation--product-get" class="swagger--panel-operation-get panel">
+                <div class="panel-heading">
+                    <div class="operation-summary">Retrieve product and pricing information associated with a premium customer subscription.</div>
+                    <h3 class="panel-title"><span class="operation-name">GET</span> <strong>/product</strong></h3>
+                </div>
+                <div class="panel-body">
+                    <section class="sw-operation-description">
+                        <p>An endpoint to retrieve details about Ambianic product.</p>
```

Comment:
@vickywane let's clarify here:
"An endpoint to retrieve details about an Ambianic premium subscription product."

Reply:
Alright. I made this adjustment